AnkiConnect.py

```diff
@@ -417,20 +417,19 @@ def deckNames(self):
             return collection.decks.allNames()
 
 
-    def launchBrowser(self, query):
+    def guiBrowse(self, query):
         browser = aqt.dialogs.open("Browser", self.window())
         browser.activateWindow()
         if query:
-            query = u'"{}"'.format(query)
+            query = unicode('"{}"'.format(query))
```

Comment:
You got the parens wrong on this one, it should be done as the code snipped I posted. Your code does a format operation on a normal string and then converts it to unicode. This will not work if `query` is unicode. What you need to do is take a unicode string and format **that**. This would be: `query = unicode('"{}"').format(query)`